doc/news/changes/minor/20200405MartinKronbichler

```diff
@@ -0,0 +1,3 @@
+Fixed: VectorTools::get_position_vector would return wrong positions for
+curved cells in the interior of the domain. This is now fixed.
+<br> (Martin Kronbichler, Doug Shi-Dong, 2020/04/05)
```

Comment:
Can you break the parentheses onto the next line? I think that the author script requires that.